examples/https/README.md

```diff
@@ -6,6 +6,11 @@ node ../../bin/webpack-dev-server.js --open --https
 
 A fake certificate is used to enable https.
 
+You can provide the following SSL options to override the fake certificate:
+
+* Certificate options e.g. `node ../../bin/webpack-dev-server.js --open --https --cacert=pathToCacert --cert=pathToCert --key=pathToKey`
```

Comment:
Could you provide a working example here? The examples are also meant to quickly test if something still works.